tidy3d/components/data/sim_data.py

```diff
@@ -1,18 +1,21 @@
 """ Simulation Level Data """
-from typing import Dict, Optional
+from typing import Dict, Optional, Union
 
 import xarray as xr
 import pydantic as pd
 import numpy as np
 
 from .monitor_data import MonitorDataType, AbstractFieldData
+from .near2far_data import Near2FarDataType
 from ..base import Tidy3dBaseModel
 from ..simulation import Simulation
 from ..boundary import BlochBoundary
 from ..types import Ax, Axis, annotate_type, Literal
 from ..viz import equal_aspect, add_ax_if_none
 from ...log import log, DataError
 
+MonitorDataType = Union[MonitorDataType, Near2FarDataType]
```

Comment:
see earlier comment, maybe this is better done in the `near2far_data.py` when `Near2FarDataType` is defined.